app/services/csv_imports/locations_service.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module CSVImports
+  class LocationsService
```

Comment:
We've got some upcoming tickets to rename all of this to `schools`. Shall we do that here or just wait for that ticket?

Reply:
Can we do the rename wholesomely? 

It somewhere between location or site.

Reply:
I think we need to discuss how deep we go with it, when we breakdown the cards.